test/test_testing.py

```diff
@@ -4640,6 +4775,7 @@ def test_set_secret_content(self):
         self.assertEqual(len(harness.charm.events), 1)
         event = harness.charm.events[0]
         self.assertIsInstance(event, ops.SecretChangedEvent)
+        assert isinstance(event, ops.SecretChangedEvent)  # Dupes previous line for type checkers.
```

Comment:
Per comment on earlier PR, I think we should *just* include the second one, which works for both tests and type checkers. Here and below.